foundations/html_css/html-foundations/html-boilerplate.md

```diff
@@ -60,25 +60,25 @@ Noticed the word `lang` here? It represents an HTML attribute which is associate
 The `<head>` element is where we put important meta-information **about** our webpages, and stuff required for our webpages to render correctly in the browser.
 Inside the `<head>`, we **should not** use any element that displays content on the webpage.
 
-#### The charset meta element
+#### The meta charset element
```

Comment:
I think part of the confusing wording in this lesson is that it is the `meta` element and the `charset` attribute. But these two get interchanged. I'd suggest updating the sub-heading to the following
```suggestion
#### The meta element
```

Reply:
Yes, I agree. "The meta element" seems to be the best solution.

For consistency, one should then probably also rephrase the other titles -- e.g. "The head element", "The title element".

Reply:
Oh, I missed this comment. In looking at the lesson this is the only one that uses "The" in the title, so I wonder if we should just change this one to be "Meta element"?

Reply:
Yes, I think that would be best.